### ISSUE TITLE
Decide which size should be used for thumbnails and stick to it

### ISSUE THREAD

**Issue by oparoz** [owncloud/core#16913]
With a devicePixelRatio of 1, the web GUI is asking for 36x36 thumbnails, but the CSS used to display those thumbnails has a hardcoded value of 32px.

Maybe some parts of the GUI need the 36x36 thumbnails and it's been designed that way in order to not have to generate both sizes. If not, we should decide whether there is room to show the 36x36 thumbnails or if we should ask for 32x32 thumbnails instead and fix all the scripts which ask for 36x36 thumbnails.

@jancborchardt  @schiesbn @PVince81 @nickvergessen

**Comment by jancborchardt** [comment]
I’m a bit confused myself now. It seems that all the thumbnails in files are 32*32, right? Do we use 36px anywhere?

**Comment by oparoz** [comment]
That's the thing. I have no idea which part uses the 36x36 thumbnails we generate.

**Comment by oparoz** [comment]
@georgehrke might know
https://github.com/owncloud/core/commit/d538a566acac35eab811b2bfa16596fb8534db0f

**Comment by georgehrke** [comment]
> d538a56

Back then, we just doubled the size of the preview, because they were too small. I don't know at what point the 36x36 slipped in.

**Comment by PVince81** [comment]
Any update ? Is this still relevant ?

**Comment by jancborchardt** [comment]
So @oparoz can you shed light on where the 36px occurs? Otherwise we can close this.

**Comment by oparoz** [comment]
Sure

```
background-image: url("/index.php/core/preview.png?file=%2F1280px-BaskaVoda13Chorvatsko.JPG&c=966918d12be5b06c25473ce18563db04&x=36&y=36&forceIcon=0"); background-size: 32px auto;
```

https://github.com/owncloud/core/blob/master/apps/files/js/filelist.js#L1385-L1399

**Comment by PVince81** [comment]
Okay... would changing these to 32 fix the issue ? I believe that in the past preview sizes were provided as attributes on the "table" element, but that might not be the case any more, or not on all tables.

**Comment by oparoz** [comment]
> Okay... would changing these to 32 fix the issue ? 

I think so. Looks like the 32px is hard-coded in 2 places, the thumbnail class and the element itself, which is unneeded.
The downside is that older ownClouders will need to regenerate 32x32px thumbnails.

**Comment by PVince81** [comment]
Regenerating thumbnails shouldn't be too bad an issue I guess

**Comment by MorrisJobke** [comment]
Fix is in #19468
